liblangutil/Exceptions.h

```diff
@@ -189,6 +191,19 @@ class Error: virtual public util::Exception
 		return "Error";
 	}
 
+    static std::optional<Severity> stringToSeverity(std::string _severity)
+	{
+        _severity = tolower(_severity);
+
+        _severity = std::regex_replace(_severity, std::regex("^ +| +$|( ) +"), "$1");
+
+        _severity.erase(_severity.begin(), std::find_if(_severity.begin(), _severity.end(), std::bind1st(std::not_equal_to<char>(), ' ')));
+
+        _severity[0] = toupper(_severity[0]);
+
+        return std::make_optional<Severity>(_severity).has_value() ? std::optional<Severity>(_severity).value() : std::nullopt;
```

Comment:
`_severity` is a `string`. You can't do `make_optional<Severity>(_severity)`.

Please make sure the code compiles locally on your machine and passes all tests.

Reply:
Will do it 👍